packages/shared/src/components/image/Image.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ImgHTMLAttributes, ReactElement, SyntheticEvent } from 'react';
+
+interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {
+  fallbackSrc?: string;
+}
+
+export const Image = ({ fallbackSrc, ...props }: ImageProps): ReactElement => {
```

Comment:
Why do we use a new `Image` element?

Reply:
It was discussed that we will be moving forward in utilizing the native lazy loading.